File: src/main/java/toast/pine/Classes/Items/ItemHandler.java
package toast.pine.Classes.Items;

import toast.pine.Events.PlayerLeftClickEvent;
import toast.pine.Events.PlayerRightCLickEvent;

public interface ItemHandler {

    void onPlayerRightClick(PlayerRightCLickEvent event);

    void onPlayerLeftClick(PlayerLeftClickEvent event);


}

File: src/main/java/pine/toast/toastrpg/core/classes/items/ItemHandler.kt
package pine.toast.toastrpg.core.classes.items

import pine.toast.toastrpg.core.events.PlayerLeftClickEvent
import pine.toast.toastrpg.core.events.PlayerRightClickEvent


interface ItemHandler {
    fun onPlayerRightClick(event: PlayerRightClickEvent)
    fun onPlayerLeftClick(event: PlayerLeftClickEvent)

}
